### ISSUE TITLE
Set up Fastly as CDN

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#830]
Fastly offered backdrop free CDN servies a while back, but we never got around to setting it up. This is a placehodler issue to figure out what's needed and push that forward.

**Comment by quicksketch** [comment]
Here's a round-up of CDN options that we should at least mention. These are based almost entirely on my own experiences, and are not impartial:

CloudFlare:

* Widely used, good APIs, nginx-based
* Very good Spam/Web-Application-Firewall capabilities, though they can frequently interfere with normal website operation.
* Intermittent, limited open-source sponsorship program: https://blog.cloudflare.com/cloudflare-open-source-your-upgrade-is-on-the-house/

Akamai:

* The original CDN. Widely used. Clunky APIs compared to newer competitors, proprietary software.
* Not sure about their WAF offerings, though Akamai tends to charge additional for each individual feature and service.
* No open-source sponsorship program that I could find.

Cloudfront

* Amazon's CDN. Widely used. Limited API capabilities and things like cache clears are notoriously difficult.
* AWS WAF is a separate product.
* AWS gives out "promotional credits" that can be used with any AWS service: https://aws.amazon.com/blogs/opensource/aws-promotional-credits-open-source-projects/

Fastly

* Widely used, good APIs, probably the most flexible choice because of customizeable VCL files, Varnish based
* Newer to Spam and Web Application Firewalls, but up and coming capabilities here. Also pluggable into other services.
* Permanent, expansive open-source sponsorship program: https://www.fastly.com/open-source/

I've used all the above on multiple projects, and Fastly is hands-down my favorite service available. Combined with the most expansive open-source program, I think this is the best candidate as a CDN for backdropcms.org

**Comment by quicksketch** [comment]
@jenlampton When did Fastly offer us services? And did we already fill out an application at https://www.fastly.com/open-source/? If not, I think that's probably the next step here.

**Comment by jenlampton** [comment]
Yes, we have an account and everything. Just never got around t setting it
up!

**Comment by jenlampton** [comment]
I have contacted Fastly today to see if our Open Source agreement needs to be updated.
